Gemfile.lock

```diff
@@ -367,6 +372,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
This might be because your version of ruby is for PC, wonder... So maybe you need this to run it on PC.   Did you ever get it running on the Mac?